debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -764,6 +825,13 @@ public ReplicationConnectionBuilder withPlugin(final PostgresConnectorConfig.Log
             return this;
         }
 
+        @Override
+        public ReplicationConnectionBuilder withReplicaIdentity(ReplicaIdentityMapper replicaIdentityMapper) {
```

Comment:
This should be unnecessary as `PostgresConnectorConfig` is passed to the class constructor.

Reply:
Removing `withReplicaIdentity` means that `replicaIdentityMapper` is always null. 
`publicationAutocreateMode` is also passed to the class constructor and `withPublicationAutocreateMode` exists